xbmc/GUIInfoManager.cpp

```diff
@@ -4026,7 +4034,7 @@ const infomap visualisation[] =  {{ "locked",           VISUALISATION_LOCKED },
 ///   \table_row3{   <b>`Fanart.Image`</b>,
 ///                  \anchor Fanart_Image
 ///                  _string_,
-///     todo
+///     Returns the fanart image for the parent TV Show
```

Comment:
not limited to TV Shows only, works for Albums as well.

Reply:
All of that is c/p from wiki

Reply:
i already mentioned that on slack when someone referred to the inline comments as "the truth".

it's not. what we have in core is a 1:1 c/p of community contributed interpretations of what those infolabels mean.  bugs, errors, incompleteness and all included.

one may wonder why we copied that info to our core in the first place...

Reply:
I'm with you, will change it to what you suggested